hacks/isolateVersion/6-updateCIWorkflows.sh

```diff
@@ -1,12 +1,9 @@
 notify "Updating CI workflows..."
 
-# 1. linkcheck: delete the file.
-rm .github/workflows/linkcheck.yaml
-
-# 2. build-docs: remove everything after the line `NODE_OPTIONS: --max_old_space_size=4096`.
+# 1. build-docs: remove everything after the line `NODE_OPTIONS: --max_old_space_size=4096`.
```

Comment:
Out of the scope of this PR, but I wonder if we could revert this NODE_OPTIONS value back to the pre-build-timeout days. 🤔  Probably not worth an investigation or even an issue since there didn't appear to be any visible consequences 🤷‍♀️

Reply:
good idea!